source/type/stop_area.cpp

```diff
@@ -46,8 +46,12 @@ void StopArea::serialize(Archive& ar, const unsigned int) {
 SERIALIZABLE(StopArea)
 
 bool StopArea::operator<(const StopArea& other) const {
-    if (name != other.name) {
-        return name < other.name;
+    std::string lower_name(name);
```

Comment:
why this is not already in lower mode ? 
Maybe during the construction we may put name var in lower directly ?